Changelogs for Digital Twin Manager UI and Microservice 1020.3.1 (#2692)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1020-1-20-remove-the-tags-from-asset-model-samples.md

---
date: ""

> title: "Remove the tags from Asset model samples"

Reviewer: ```suggestion
title: "Remove the tags from asset model samples"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1653"
version: "1020.1.20"
---

> Previously SAG and DTM tags has been added to the samples in 'Asset model samples' page. Now it has been removed.

Reviewer 2: Previously, SAG and DTM tags were added to the samples on the 'Asset Model Samples' page. These tags have now been removed.

> Previously SAG and DTM tags has been added to the samples in 'Asset model samples' page. Now it has been removed.

Reviewer: ```suggestion
Previously, SAG and DTM tags have been added to the samples in the **Asset model samples** page. These tags have now been removed.
```

> Previously, SAG and DTM tags have been added to the samples in the **Asset model samples** page. These tags have now been removed.

Reviewer: What are SAG and DTM tags? Is this clear for the user? Or does it need more explanation?

Reviewer 3: @pavanshivajirao, @Vagdevi1998  i think this can be removed from the changelog, might not be relevant in changelog and it was done due to the carveout.

Reviewer 4: Removed
